test/e2e/chrome-devtools-workspace/chrome-devtools-workspace-base-path.test.ts

```diff
@@ -0,0 +1,26 @@
+import * as path from 'path'
+import { nextTestSetup } from 'e2e-utils'
+
+describe('chrome-devtools-workspace basePath', () => {
+  const { isNextDev, next } = nextTestSetup({
+    files: path.join(__dirname, 'fixtures', 'base-path'),
+  })
+
+  it('should be able to connect to Chrome DevTools in dev with a configured basePath', async () => {
+    const devtoolsResponse = await next.fetch(
+      // Chrome DevTools always requests `/`.
+      '/.well-known/appspecific/com.chrome.devtools.json'
+    )
```

Comment:
The need some way to tell Chrome DevTools to request it from the configured `basePath` and then only handle requests targeting that endpoint.